java/runtime/src/main/java/io/ray/runtime/gcs/GlobalStateAccessor.java

```diff
@@ -0,0 +1,67 @@
+package io.ray.runtime.gcs;
+
+import com.google.common.base.Preconditions;
+import java.util.List;
+
+/**
+ * `GlobalStateAccessor` is used for accessing information from GCS.
+ *
+ **/
+public class GlobalStateAccessor {
+  // NOTE(lingxuan.zlx): Native pointer is singleton in gcs state accessor, which means it can not
+  // be changed in cluster when redis or other storage accessor is fixed.
+  private static Long globalStateAccessorNativePtr = 0L;
```

Comment:
```suggestion
  private static Long globalStateAccessorNativePointer = 0L;
```
avoid using abbreviations.